doc/GIFsGuide.md

```diff
@@ -0,0 +1,15 @@
+# PowerPointLabs Website GIFs Guide
+
+Follow these instructions to produce consistent GIFs to be displayed in the PowerPointLabs website
+1. Create animations using PowerPoint
+1. With the animations, go to `Export` -> `Create a Video`
+   1. Quality: Low/ Standard (Smallest file size and lowest quality)
+   1. Recordings: Use Recorded Timings and Narrations
+   1. Seconds spent on each slide: 00.50
+1. Click `Create Video`
+1. [Download](https://www.google.com/url?q=https%3A%2F%2Ffree_video_to_gif_converter.en.softonic.com%2F) a MP4-to-GIF Converter
```

Comment:
Why not use the direct URL? Also better to open in a new tab (if not already)

Reply:
noted and included